hoon/reference/stdlib/2b.md

```diff
@@ -564,6 +564,11 @@ the final value of the accumulator.
 (To "slam" means to call a gate and give it a sample/samples. In this instance,
 `a` is the list of samples that are given to the gate b.)
 
+The initial value of the accumulator is `+>+.$`. This can occasionally produce
```

Comment:
```suggestion
The initial value of the accumulator is the bunt of `b`'s second argument. This can occasionally produce
```

Something like this would be much more clear. For the lark, you would've wanted `+<+`, in any case.